.github/workflows/ci.yml

```diff
@@ -118,29 +118,11 @@ jobs:
         with:
           save-if: ${{ github.event_name != 'merge_group' }}
 
-      - name: cargo clippy (windows)
-        run: cargo hack clippy --workspace ${{ env.NO_WINDOWS_PKGS }} --locked --optional-deps --each-feature -- -D warnings
-        if: contains(matrix.os, 'windows')
-
-      - name: cargo clippy (auxiliary) (windows)
-        run: cargo hack clippy --workspace ${{ env.NO_WINDOWS_PKGS }} --locked --optional-deps --each-feature --tests --benches --examples -- -D warnings
-        if: contains(matrix.os, 'windows')
-
-      - name: cargo clippy (macos)
-        run: cargo hack clippy --workspace ${{ env.NO_MACOS_PKGS }} --locked --optional-deps --each-feature -- -D warnings
-        if: contains(matrix.os, 'macos')
-
-      - name: cargo clippy (auxiliary) (macos)
-        run: cargo hack clippy --workspace ${{ env.NO_MACOS_PKGS }} --locked --optional-deps --each-feature --tests --benches --examples -- -D warnings
-        if: contains(matrix.os, 'macos')
-
-      - name: cargo clippy (ubuntu)
-        run: cargo hack clippy --workspace ${{ env.NO_UBUNTU_PKGS }} --locked --optional-deps --each-feature -- -D warnings
-        if: contains(matrix.os, 'ubuntu')
+      - name: cargo clippy
+        run: cargo hack clippy --workspace ${{ fromJson(env.EXCLUDE_PKGS)[runner.os] }} --locked --optional-deps --each-feature -- -D warnings
```

Comment:
What happens if we get a `null` from here? My first thought is that I want an error, but thinking about it further, the empty string would be better - CI would fail if we had a typo, because the package wouldn't be excluded.

Reply:
I tested it and referencing an undefined property converts into an empty string. So yeah, the CI would fail because by design it would only pass if there is no mistake.